Kimseho_assignment/app.js

```diff
@@ -46,6 +55,34 @@ app.get("/posts", async (req, res) => {
   );
 });
 
+
+app.post("/login", async (req, res) => {
+  const { email, password } = req.body;
+  try {
+    const [checkUser] = await myDataSource.query(
+      `SELECT 
+        email,
+        password
+    FROM users
+    WHERE email = ?`,
+      [email]
+    );
+
+    const checkHash = (password, hashedPassword) => {
+      return bcrypt.compare(password, hashedPassword);
+    };
+
+    if (await checkHash(password, checkUser["password"])) {
+      const jwtToken = jwt.sign(payLoad, checkUser["password"]);
+      return res.status(201).json({ accessToken: jwtToken });
+    } else {
+      return res.status(404).json({ message: "Invalid User" });
+    }
+  } catch (err) {
+    console.log(err);
+    return res.status(409).json({ message: "email wrong" });
```

Comment:
세호님 ! HTTP status code중 409는 서버와 충돌이 발생했음을 의미하는 코드입니다!
잘못된 요청의 경우 400 code를 사용해 보시는 건 어떨까요?
```suggestion
  } catch (err) {
    console.log(err);
    return res.status(400).json({ message: "email wrong" });
```